libs-scala/direct-execution-context/src/main/scala/com/digitalasset/dec/DirectExecutionContext.scala

```diff
@@ -7,14 +7,17 @@ import org.slf4j.LoggerFactory
 
 import scala.concurrent.ExecutionContext
 
-// Starting from Scala 2.13 this can deleted and replaced by `parasitic`
```

Comment:
Probably want to keep this comment.

Reply:
Yeah, I removed it before I changed it so that it could have different messages based on the context, but I'll roll back. Thanks!

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/2ff3228269431a9367a8777279454e4fbce060f4